Jobs/PrGate.yml

```diff
@@ -110,3 +111,15 @@ jobs:
         do_pr_eval: ${{ parameters.do_pr_eval }}
         tool_chain_tag: ${{ parameters.tool_chain_tag }}
         install_tools: ${{ not(parameters.container_build) }}
+        extra_build_args: CODE_COVERAGE=TRUE
+
+  - job: PublishCoverage
+    dependsOn: Build
+    workspace:
+      clean: all
+
+    pool:
+      vmImage: 'windows-2022'

```

Comment:
If this doesn't have VS toolchain dependencies (it does not) then I suggest to use "`windows-latest`". It will be one less place to fall behind and cause problems in the future when "`windows-2022`" has brown outs and is eventually deprecated.